cmake/helpers.cmake

```diff
@@ -63,6 +63,8 @@ function(add_ur_target_compile_options name)
             -fPIC
             -Wall
             -Wpedantic
+            -Wempty-body
+            -Wunused-parameter
```

Comment:
While I think adding `-Wunused-paramter` is a good idea, perhaps that should be a separate PR since its exposed additional warnings.